src/config/store.ts

```diff
@@ -6,6 +6,39 @@ import defaults from './defaults';
 import { DefaultPresetList, type Preset } from '@/plugins/downloader/types';
 
 const migrations = {
+  '>=3.2.2'(store: Conf<Record<string, unknown>>) {
```

Comment:
```suggestion
  '>=3.3.0'(store: Conf<Record<string, unknown>>) {
```
If the plugin is added, we'll bump the minor version.